src/Features/VisualBasic/GenerateType/VisualBasicGenerateTypeService.vb

```diff
@@ -90,10 +90,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.GenerateType
                 document As SemanticDocument, simpleName As SimpleNameSyntax, cancellationToken As CancellationToken, ByRef generateTypeServiceStateOptions As GenerateTypeServiceStateOptions) As Boolean
             generateTypeServiceStateOptions = New GenerateTypeServiceStateOptions()
 
-            If simpleName.GetAncestorOrThis(Of ImportsStatementSyntax)() IsNot Nothing Then
```

Comment:
There were arguments on both sides before the C# feature.  I would say 99% of the time someone is mistyping an existing namespace instead of wanting a type to be generated. Now that C# does this for using static it seems like a good idea to introduce parity.